src/actions.cr

```diff
@@ -7,14 +7,14 @@ module DppmRestApi::Actions
   extend self
   include RouteHelpers
 
+  ROUTE_MODULES    = [Pkg, App, Service, Src, User, Users, Groups]
```

Comment:
This constant doesn't seem to be useful, as-is

Reply:
Yes it was supposed to be for the other thing.